v2/dtos/common/config.go

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+// ConfigRequest defines the request associated with the /config endpoint.
+// This object and its properties correspond to the ConfigRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/ConfigRequest
+type ConfigRequest struct {
+	RequestID string `json:"requestId" validate:"required,uuid"`
+}
+
+// ConfigResponse defines the configuration for the targeted service.
+// This object and its properties correspond to the ConfigResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/ConfigResponse
+type ConfigResponse struct {
+	BaseResponse `json:",inline"`
+	Config       string `json:"config"`
```

Comment:
would this be better as an `interface{}` ?

Reply:
@rsdmike Would you please explain which part should be an `interface{}` and the reason for it? Thanks

Reply:
@lindseysimple , Each services conifg struct is different. Making this an `interface{}` allows us to populate the field with any services config struct and avoid the `embedded JSON`

Reply:
```
{
     "statusCode":200,
     "config": "{\"property\":\"field\"}" //an embedded json version of the config -- not so pretty
}
```
vs
```
{
     "statusCode":200,
     "config": {"property": "field"}   //properly a part of the json
}
```

Reply:
@lenny-intel @rsdmike , thanks for the explanation. Have updated `Config` to `Interface{}`. 
Would you please review again? Thanks.